CreeDictionary/API/search/core.py

```diff
@@ -42,24 +42,22 @@ def sorted_results(self) -> list[types.Result]:
 
     def presentation_results(self) -> list[presentation.PresentationResult]:
         results = self.sorted_results()
-        try:
-            prefetch_related_objects(
-                [r.wordform for r in results],
-                "lemma__definitions__citations",
-                "definitions__citations",
-            )
-        except AttributeError:
-            # This happens rarely, but is reproducible with the test suite:
-            # “'RelatedManager' object has no attribute 'citations'.” I think
-            # this is a django bug? It tries to look up a field from Definition
-            # on a DictionarySource object. Passing on the exception just means
-            # that some searches will run more slowly. To revisit when upgrading
-            # to Django 3.
```

Comment:
So this bug no longer exists?

Reply:
I’m not 100% sure the bug is gone, but it used to be triggered during the unit tests. After upgrading Django that doesn’t happen anymore. That’s good enough for me.